hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationService.java

```diff
@@ -1102,13 +1115,17 @@ public void run() {
                     backPressureService.cleanup();
```

Comment:
in case of shutdown, did we really need to wait ```scanHandleOperationTimeout```to finish its work. In case of shutdown, it should break ```for``` loop in itself